### PR TITLE
remove print media at rules

### PR DESCRIPTION
**Pull request by peterbe**
I took the fixture you started in https://github.com/peterbe/minimalcss/pull/88 and copied it into this. Then I made the necessary code change so it actually makes the minimal CSS remove the `@media print {...}` block.

### CODE REVIEW

**Comment by peterbe** [comment]
I hope this doesn't complicate things for your tests PR. If so, this can wait.

File: tests/main.test.js [hunk 1]
```diff
@@ -63,3 +63,9 @@ test('handles 404 CSS file', async () => {
     expect(e.message).toMatch('404 on')
   }
 })
+
+test('media queries print removed', async () => {
```
Comment (stereobooster) [inline tests/main.test.js]: I have test (in big PR) that shows that print media queries stays.

Comment (peterbe) [reply tests/main.test.js]: I know and I think that's "wrong". I.e. the `@media print ...` blocks *should* be removed. 
I suggest you remove "your version" and then merging with this PR will be easier for you.

Comment (stereobooster) [reply tests/main.test.js]: I mean my tests are stating facts about current code - exposing right and wrong cases, there are more wrong cases. As soon as you fix code you update tests. I just referred to the fact that there will be merge conflict nothing else

**Comment by stereobooster** [comment]
this needs update in changelog

**Pull request merged**